### DISCUSSION TITLE
Feature/Change requests for SpiceSharpParser

### DISCUSSION THREAD

**Discussion by marcin-golebiowski** [OpenSemi-de/ACDCs#45] [General]
Hi,

I see that you are using the library I've been working on some time ago.
I would like to kindly ask about your needs or feature requests regarding SpiceSharp-Parser library.
I would be happy to fix, improve, add or change something on your request.

Best regards,
Marcin Gołębiowski

**Comment by OpenSemi-de** [comment]
Hi there,

sorry for the late reply. I had to wait to get the simulation running to see if Spice# is usable in my case. This point was reached yesterday, so the next phase for the app starts. First off all i want to thank you for the nice parser - it is doing its work very good. I had some problems that i ignore actually - if you like i can give you the bug reports. 

Regards,
Stephan Baltzer

**Reply by marcin-golebiowski** [reply]
Hi,

Sorry for the late reply. 
Please submit the bug reports for SpiceSharpParser.
That will help a lot.

Best regards,
Marcin Gołębiowski
